spec.md

```diff
@@ -936,6 +976,8 @@ message NodePublishVolumeResponse {
 
 A Node Plugin MUST implement this RPC call.
 This RPC is a reverse operation of `NodePublishVolume`.
+This RPC MUST remove any mounts setup by the corresponding  `NodePublishVolume`.
+This Plugin SHALL assume that this RPC will be executed at least once for each successful `NodePublishVolume` call.
```

Comment:
I think we should call out that for each `target_path`, there should be at least one call to `NodePublishVolume` by the CO.